examples/benchmark/sink/src/main.rs

```diff
@@ -0,0 +1,67 @@
+use dora_node_api::{self, DoraNode};
+use std::{
+    fmt::Write as _,
+    time::{Duration, Instant},
+};
+
+fn main() -> eyre::Result<()> {
+    let mut node = DoraNode::init_from_env()?;
+    let inputs = node.inputs()?;
+
+    let mut current_size = 0;
+    let mut n = 0;
+    let mut start = Instant::now();
+    let mut latencies = Vec::new();
+
+    let mut summary = String::new();
+
+    while let Ok(input) = inputs.recv() {
+        match input.id.as_str() {
+            "message" => {
+                let data = input.data();
+
+                if data.len() != current_size {
+                    if n > 0 {
+                        record_results(start, current_size, n, latencies, &mut summary);
+                    }
+                    current_size = data.len();
+                    n = 0;
+                    start = Instant::now();
+                    latencies = Vec::new();
+                }
+                n += 1;
+                latencies.push(
+                    input
+                        .metadata
+                        .timestamp()
+                        .get_time()
+                        .to_system_time()
+                        .elapsed()?,
```

Comment:
So could it be that for very small messages, the message is being sent faster than it can be processed and therefore you have an error in latency time? Maybe by either waiting between sending or using a tick as input, the small input may have better accuracy.

Reply:
Done in https://github.com/dora-rs/dora/pull/170/commits/6dc0d58c8df167c0c65d7f84422eadc634b486e9